processor/sourcemap/package_tests/processor_test.go

```diff
@@ -20,7 +20,7 @@ func TestSourcemapProcessorOK(t *testing.T) {
 
 // ensure invalid documents fail the json schema validation already
 func TestTransactionProcessorValidationFailed(t *testing.T) {
```

Comment:
not introduced here, but should these tests be named `TestSourcemapProcessorValidation`... ?